quinn/src/notify.rs

```diff
@@ -0,0 +1,89 @@
+use std::sync::{Arc, Mutex};
+use std::task::{Context, Waker};
+
+use slab::Slab;
+
+/// Broadcasts an event to any number of waiters
+#[derive(Clone)]
+pub struct NotifyOwned {
+    shared: Arc<Mutex<Shared>>,
+}
+
+struct Shared {
+    version: u64,
```

Comment:
I feel like calling this `epoch` might better evoke how it's used? Perhaps the field here could get some docs on how/why it's supposed to be used?

Reply:
Mooted by adoption of `tokio::sync::Notify`.